### ISSUE TITLE
Suddenly can't connect to cluster

### ISSUE THREAD

**Issue by nhatsangvn** [lensapp/lens#3733]
**Describe the bug**
I was using my cluster and suddenly the bug happened
**To Reproduce**
1.  Connect to my cluster

**Expected behavior**
Connect mnormally

**Screenshots**
![image](https://user-images.githubusercontent.com/20770118/132013495-ef9ba8a7-79c9-4673-8d43-32ee31160760.png)

**Environment (please complete the following information):**
- Lens Version: 4.2.4
- OS: OSX 10.15.7
- Installation method (e.g. snap or AppImage in Linux): using .dmg file

**Logs:**
When I connect to my cluster
```
App crash at http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/?lens-namespaces=default

Component stack:
 at http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:7553390
    at div
    at div
    at div
    at dD (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:8274124)
    at div
    at tF (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:8302823)
    at div
    at div
    at cF (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:8308806)
    at footer
    at div
    at pU (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:8514060)
    at cc (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:7683374)
    at t (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:223994)
    at BU (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:8515564)
    
Error stack:
TypeError: Cannot read property 'containers' of undefined
    at Of.getContainers (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:7899747)
    at http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:8292941
    at http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:7553411
    at http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:7553024
    at ce (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:12079)
    at e.track (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:23443)
    at http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:7553000
    at http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:7552837
    at G (http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:7552974)
    at http://42b8157f-d51d-4f57-b772-653ab6042113.localhost:49554/build/Lens.js:2:7553390   
```

**Comment by jim-docker** [comment]
Can you upgrade to version 5 to see if the issue still happens?

**Comment by sergiomcalzada** [comment]
Similar issue here
* Windows 10
* Lens 5.1.3-latest.20210722.1


```
Component stack:


    at http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:7000:16
    at div
    at div
    at div
    at InfoPanel (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:540039:9)
    at div
    at Logs (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:542060:9)
    at div
    at div
    at Dock (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:542472:9)
    at div
    at div
    at MainLayout (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:488069:9)
    at ErrorBoundary (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:487530:47)
    at ErrorBoundary (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:487639:36)
    at Router (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:20003:30)
    at App (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:543551:9)
    at ThemeProvider (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:568264:24)
```

```
TypeError: Cannot read property 'namespace' of undefined
    at pods_api_Pod.getNs (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:511980:30)
    at pods_api_Pod.getOwnerRefs (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:512011:32)
    at http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:541299:30
    at http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:7000:49
    at http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:6964:25
    at trackDerivedFunction (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:1881:18)
    at Reaction.track (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:2409:18)
    at useObserver (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:6962:14)
    at http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:7000:16
    at Ch (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450238:137)
    at li (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450261:154)
    at ki (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450259:169)
    at ii (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450258:178)
    at ck (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450355:126)
    at bk (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450331:347)
    at ak (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450331:278)
    at Tj (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450331:138)
    at Lj (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450324:163)
    at Jg (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450318:175)
    at lk (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450366:27)
    at http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450370:301
    at Xj (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450325:244)
    at tk (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450370:287)
    at exports.render (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:450377:69)
    at bootstrap (http://04a5db29fb76508fb1d928219e9b5e1d.localhost:52765/build/Lens.js:578417:32)
````

**Comment by nhatsangvn** [comment]
> Can you upgrade to version 5 to see if the issue still happens?

It seems working now. Thanks you. Hope it wont happen again.

**Comment by sergiomcalzada** [comment]
Uninstalled, deleted all _Lens references_ under "AppData" folders, and installed the latest 5.2.0-latest.20210908.1.
Now it works

**Comment by Nokel81** [comment]
Fixed by #3361
